sequential/seq2pat.py

```diff
@@ -214,6 +226,23 @@ def has_lower_bound(self):
     def has_upper_bound(self):
         return self.upper_bound is not None
 
+    def check_satisfaction(self, value):
+        res = [True]
```

Comment:
say a word on why this needs to start from [True] by default

Reply:
I understand. What I meant is to add a comment on the code for why so, for the next person looking at this

Reply:
Have added the comment and also have the logic simplified as suggested by @dorukkilitcioglu .